recipe/meta.yaml

```
@@ -53,25 +66,98 @@ requirements:
+{% set tests_to_skip = tests_to_skip + " or test_transformer_cross_self_attention" %}
+{% set tests_to_skip = tests_to_skip + " or test_transformer_pointer_generator" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_dicts" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_no_vepoch" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_src_tgt_dict_spec" %}
+{% set tests_to_skip = tests_to_skip + " or test_activation_checkpointing_does_not_change_metrics" %}
+{% set tests_to_skip = tests_to_skip + " or test_activation_offloading_does_not_change_metrics" %}
+{% set tests_to_skip = tests_to_skip + " or test_mask_for_xformers" %}
+
+# requires examples, but skipping examples import due to segmentation fault from the size of examples
+{% set tests_to_skip = tests_to_skip + " or test_laser_lstm" %}
+{% set tests_to_skip = tests_to_skip + " or test_laser_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_masked_lm" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_regression_multiple" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_regression_single" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_sentence_prediction" %}
+{% set tests_to_skip = tests_to_skip + " or test_r4f_roberta" %}
+{% set tests_to_skip = tests_to_skip + " or test_waitk_checkpoint" %}
+{% set tests_to_skip = tests_to_skip + " or test_multilingual_translation_latent_depth" %}
+{% set tests_to_skip = tests_to_skip + " or test_roberta_incremental_decoder" %}
 
 test:
+  source_files:
+    - tests
+    - scripts
+    - fairseq
+  requires:
+    - pytest
+    - hypothesis
+    - expecttest
+    - pip
+    - tensorboardx
   imports:
     - fairseq
     - fairseq.benchmark
   commands:
+    - pip check
     - fairseq-eval-lm --help
     - fairseq-generate --help
     - fairseq-interactive --help
     - fairseq-preprocess --help
     - fairseq-score --help
     - fairseq-train --help
     - fairseq-validate --help
+    - pytest -vv .  --ignore=tests/speech_recognition/test_collaters.py --ignore=tests/speech_recognition/test_cross_entropy.py --ignore=tests/speech_recognition/test_data_utils.py --ignore=tests/speech_recognition/test_vggtransformer.py -k "not({{ tests_to_skip }})"
 
 about:
   home: https://github.com/pytorch/fairseq
   summary: Facebook AI Research Sequence-to-Sequence Toolkit
+  description: |
+    Fairseq(-py) is a sequence modeling toolkit that allows researchers and 
+    developers to train custom models for translation, summarization, language
+    modeling and other text generation tasks.
   license: MIT
   license_file: LICENSE
+  license_family: MIT
+  dev_url: https://github.com/facebookresearch/fairseq
+  doc_url: https://fairseq.readthedocs.io/en/latest/
```

```suggestion
  doc_url: https://fairseq.readthedocs.io/
```
To address the linter complains